packages/p2p-v2/src/hooks/__tests__/useModalManager.spec.tsx

```diff
@@ -0,0 +1,332 @@
+import * as React from 'react';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router-dom';
+import { act } from '@testing-library/react';
+import { renderHook } from '@testing-library/react-hooks';
+import useDevice from '../useDevice';
```

Comment:
for this we should use the useDevice hook from deriv-com/ui library, not the one in hooks anymore